src/main/java/uk/gov/hmcts/cmc/claimstore/documents/content/FullDefenceResponseContentProvider.java

```diff
@@ -0,0 +1,79 @@
+package uk.gov.hmcts.cmc.claimstore.documents.content;
+
+import com.google.common.collect.ImmutableMap;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.documents.content.models.EvidenceContent;
+import uk.gov.hmcts.cmc.domain.models.PaymentDeclaration;
+import uk.gov.hmcts.cmc.domain.models.TimelineEvent;
+import uk.gov.hmcts.cmc.domain.models.evidence.DefendantEvidence;
+import uk.gov.hmcts.cmc.domain.models.response.DefenceType;
+import uk.gov.hmcts.cmc.domain.models.response.DefendantTimeline;
+import uk.gov.hmcts.cmc.domain.models.response.FullDefenceResponse;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static java.util.Objects.requireNonNull;
+import static uk.gov.hmcts.cmc.claimstore.utils.Formatting.formatDate;
+
+@Component
+public class FullDefenceResponseContentProvider {
+
+    public Map<String, Object> createContent(FullDefenceResponse fullDefenceResponse) {
+        requireNonNull(fullDefenceResponse);
+        Map<String, Object> content = new HashMap<>();
+
+        List<TimelineEvent> events = null;
+        List<EvidenceContent> evidences = null;
+        String timelineComment = null;
+        String evidenceComment = null;
+
+        content.put("responseDefence", fullDefenceResponse.getDefence().orElse(null));
```

Comment:
There are some similarities between both providers but no point removing code duplicates as the whole concept of providers is going to disappear as it just does not work well.